scaladoc/src/dotty/tools/scaladoc/renderers/Renderer.scala

```diff
@@ -101,7 +101,7 @@ abstract class Renderer(rootPackage: Member, val members: Map[DRI, Member], prot
     val all = navigablePage +: redirectPages
     // We need to check for conflicts only if we have top-level member called docs
     val hasPotentialConflict =
-      rootPackage.members.exists(m => m.name.startsWith("_docs"))
+      rootPackage.members.exists(m => m.name.startsWith("docs"))
```

Comment:
Isn’t it the case only if `apiSubdirectory` is `false`? Also, would a package named `blog` cause a conflict too?

Reply:
`blog` won't cause a conflict because it's rendered under `docs`

This place is just a heuristics that informs user about potential conflict. Even so, of course,  checking the apiSubdirectory flag seems sensible.